accepted/0009-audio-busses.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2020-01-17
+- RFC PR: #9
+- Mantis Issue: none
+
+- Related PRs:
+	1. https://github.com/obsproject/obs-studio/pull/1045
+
+# Summary
+
+Add the ability for users to output audio monitoring to multiple devices.
+
+# Motivation
+
+Users want to be able to output to multiple audio devices. For example they could monitor the audio with headphones and also output the audio to speakers at the same time. They also don't have to use other software for this.
+
+# Detailed design
+
+## User UX
+
+- In https://github.com/obsproject/obs-studio/pull/1253, a headphone checkbox is added to the volume control widget (needs to be merged).
+- In a right click menu for the headphone checkbox, add items for Bus A and Bus B where users would click to enable the audio busses.
```

Comment:
I don't like this, as this means that toggling headphone would also toggle busses A and B. The main reason to have audio busses is so that some sources goes to only one bus, some will go to another, and some will go to all of them.